components/Chips/src/MDCChipCollectionViewCell.m

```diff
@@ -18,11 +18,14 @@
 
 #import "private/MDCChipView+Private.h"
 
-@implementation MDCChipCollectionViewCell
+@implementation MDCChipCollectionViewCell {
+  BOOL _isSelected;
+}
 
 - (instancetype)initWithFrame:(CGRect)rect {
   if (self = [super initWithFrame:rect]) {
     _chipView = [self createChipView];
+    _isSelected = NO;
```

Comment:
Nit: This is the default values and [instance variables should not be initialized to their defaults](http://google.github.io/styleguide/objcguide.html#initialization).